web/app.js

```diff
@@ -2518,7 +2518,7 @@ function webViewerPresentationMode() {
   PDFViewerApplication.requestPresentationMode();
 }
 function webViewerSwitchAnnotationEditorMode(evt) {
-  PDFViewerApplication.pdfViewer.annotationEditorMode = evt.mode;
+  PDFViewerApplication.pdfViewer.setAnnotationEditorMode(evt.mode, evt.editId);
```

Comment:
Please keep this as a getter.
```suggestion
  PDFViewerApplication.pdfViewer.annotationEditorMode = evt;

Reply:
Also, you need to change https://github.com/mozilla/pdf.js/blob/c33e6ceb03fd3e824dd1bd4c88bec78f5ccd1657/web/pdf_presentation_mode.js#L178
into
```javascript
this.pdfViewer.annotationEditorMode = { mode: AnnotationEditorType.NONE };
```